module-apps/application-messages/windows/NewMessage.cpp

```diff
@@ -46,14 +46,40 @@ namespace gui
             state.reset();
         }
 
-      private:
+      protected:
         std::unique_ptr<gui::TextBackup> state;
     };
 
+    class NewMessageWindow::RecipientMemento : public NewMessageWindow::MessageMemento
+    {
+      public:
+        void setState(const gui::Text *_state, const std::shared_ptr<ContactRecord> _contact)
```

Comment:
I don't think we use code such naming convention which starts with _. Consider rename the variables to state and contact

Reply:
Additionally, I think that with such a convention members are named with _  and arguments without it